build.go

```diff
@@ -261,8 +262,14 @@ func main() {
 	args := []string{
 		"-tags", strings.Join(buildTags, " "),
 		"-ldflags", fmt.Sprintf(`-s -X main.version %q -X main.compiledAt %q`, version, compileTime),
-		"-o", "restic", "github.com/restic/restic/cmd/restic",
 	}
+	if runtime.GOOS != "windows" {
```

Comment:
This could be much shorter, by setting the output file name depending on `GOOS` ;)